server/schemas/schemaModel.js

```diff
@@ -28,32 +28,26 @@ module.exports = {
         //console.debug("findOneAsync args:", arguments);
         if (schema) {
           error = {message: "Schema already exists"};
-          helpers.errorHandler(error, request, response, next);
-          return;
+          next(error, false);
         } else {
-          Model.createAsync({
+          return Model.createAsync({
```

Comment:
I see a create in mongoose [1](http://mongoosejs.com/docs/api.html#model_Model.create) but no createAsync, does mongoose have something where you can add async to the end of functions?